frontend/src/components/ReviewPartnerDialog.js

```diff
@@ -31,17 +38,16 @@ const ReviewPartnerDialog = ({ dialogOpen, handleCloseDialog }) => {
     navigate(homeUrl)
   }
 
-  const handleSubmitReview = () => {
-    /*
-      TODO: Integrate backend of review service
+  const handleSubmitReview = async () => {
+    const FIELDS = [
+      codeCorrectnessValue,
+      codeDesignValue,
+      codeStyleValue,
+      communicationValue,
+      timeManagementValue,
+    ]
 
-      Values are represented as follows:
-        Code Correctness: codeCorrectnessValue
-        Code Design: codeDesignValue
-        Code Style: codeStyleValue
-        Communication: communicationValue
-        Time Management: timeManagementValue
-    */
+    await createReviewStats(user._id, partneruuid, FIELDS)
```

Comment:
```suggestion
    await createReviewStats(partneruuid, user._id, FIELDS)
```
I think the arguments should be in this order right 😬